src/arch/common/cc-dpcpp.sh

```diff
@@ -0,0 +1,25 @@
+CMK_CPP_CHARM="cpp -P"
+CMK_CPP_C="icx$CMK_COMPILER_SUFFIX"
+CMK_CC="icx$CMK_COMPILER_SUFFIX"
+CMK_LD="icx$CMK_COMPILER_SUFFIX"
+CMK_CXX="dpcpp$CMK_COMPILER_SUFFIX"
+CMK_LDXX="dpcpp$CMK_COMPILER_SUFFIX"
+
+CMK_CPP_C_FLAGS="-E"
+
+if [ "$CMK_COMPILER" = "msvc" ]; then
+  CMK_AR='ar q'
+  CMK_LIBS='-lws2_32 -lpsapi -lkernel32'
+  CMK_SEQ_LIBS="$CMK_LIBS"
+
+  CMK_NATIVE_CC="$CMK_CC"
+  CMK_NATIVE_LD="$CMK_LD"
+  CMK_NATIVE_CXX="$CMK_CXX"
+  CMK_NATIVE_LDXX="$CMK_LDXX"
+fi
```

Comment:
Will this block ever get hit? Is it meaningful since line 24 sets it to `icx` anyway?

Reply:
Maybe on Windows (I think that's where `msvc` is used), this file has been mostly copied over from `cc-clang.sh` since `dpcpp` uses `clang` underneath.
We can remove it for now and revisit it when we have the need for dpcpp builds on Windows.